frontend/common/constants.ts

```diff
@@ -441,6 +441,10 @@ export default {
       'TRAITS_ID': 150,
     },
   },
+  getFlagsmithSDKUrl: () =>
+    Utils.isSaas() || Project.api.startsWith('https://api.flagsmith.com')
+      ? Project.flagsmithClientEdgeAPI
+      : Project.api,
```

Comment:
I don't really know what the solution is here, but we need to account for when `Project.api` is just `/api/v1` as per the above screenshot.

Reply:
Ah I didn't expect that to be relative, should be fixed

Reply:
If it helps, this is how I solved a similar problem for displaying SAML URLs based on absolute or relative API URLs: https://github.com/Flagsmith/flagsmith/blob/bb7849c799fff98c1750adf1d5a057479994d41b/frontend/web/components/modals/CreateSAML.tsx#L58